### ISSUE TITLE
Fix this when generating classlist CSV

### ISSUE THREAD

**Issue by mkijowski** [wrightedu/CSE-Discord-Bot#398]
### remove following:

```csv
CEG 1900,,CEG 1900,Internship,"#ceg1900,0#Student Voice,2#TA Voice"
CS 1900,,CEG 1900,Internship,
```

### edit following:

```csv
CEG 5350,,CEG 5350,Operating Systems Internals and Design,"#ceg5350,0#Student Voice,2#TA Voice"
    becomes
CEG 5350,,CEG 2350,Operating Systems Internals and Design,

CEG 6424,,CEG 6424,Security Attacks & Defenses,"#ceg6424,0#Student Voice,2#TA Voice"
    becomes
CEG 6424,,CEG 4424,Security Attacks and Defenses,
```

#### Crosslisting issues: Names differ

* CEG 5350 crosslisted with CEG 4350
* CEG 6424 crosslisted with CEG 4424
* EE 2011 crosslisted with EE 2010
* EE 4910 crosslisted with CEG 4980
* EE 4920 crosslisted with CEG 4981
* EE 7840 corsslisted with CEG 7550

**Comment by JakeHamblin** [comment]
Will look at modifying the script to automatically handle this.

**Comment by JakeHamblin** [comment]
Similar issue to #405 and fixed in #406

**Comment by JakeHamblin** [comment]
Closed as a result of #406 completion.
